src/Functions/Altinn.Auth.AuditLog.Functions/Models/AuthorizationEvent.cs

```diff
@@ -18,7 +18,7 @@ public class AuthorizationEvent
         /// <summary>
         /// Date, time of the authorization event. Set by producer of logevents
         /// </summary>
-        public DateTime Created { get; set; }
+        public DateTimeOffset Created { get; set; }
```

Comment:
This needs to be `DateTimeOffset?` - else you have no way of knowing if a value was provided.